CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # bedrock-ledger ChangeLog
 
+## 11.0.0 -
+
+### Changed
+- **BREAKING**: Use `bedrock-mongodb` 7.x-8.x.
+- Updated usage of MongoDB api to version 3.5.
+- Use `collection.{updateOne, updateMany}` over `collection.update`.
+- Pass `{projection}` to various methods as an option.
+
```

Comment:
We will need to add an update for the new consensus worker scheduler.

Reply:
Added.